lib/app/file.js

```diff
@@ -1,10 +1,15 @@
 'use strict';
 
-var ArrayStore = require('fh-wfm-mediator/lib/array-store');
-
-var files = [];
+var SimpleStore = require('fh-wfm-simple-store');
 
 module.exports = function(mediator) {
-  var arrayStore = new ArrayStore('file', files);
-  arrayStore.listen('cloud:', mediator);
+  var fileStore = new SimpleStore('file');
+
+  fileStore.list().then(function(res) {
+    if (!res || res.length === 0) {
```

Comment:
Can you add a comment here covering what is being checked and why. 

It should probably be noted that it should be removed in production applications as they are likely to be starting with 0 of everything.